crowdsec-docs/docs/bouncers/haproxy.mdx

```diff
@@ -167,7 +167,7 @@ backend captcha_verifier
 
 # define a backend for crowdsec to allow DNS resolution
 backend crowdsec
- server crowdsec crowdsec:8080 check
+ server crowdsec 127.0.0.1:8080 check
```

Comment:
Crowdsec is not necessarily on the same machine that the HA Proxy. It is ok to change it to 127.0.0.1 but can you add a comment to mention that this should be the URI of the crowdsec Local API please?